#! /bin/sh /usr/share/dpatch/dpatch-run
## 05_ubuntu_use_pkg_acq_run.dpatch by  <egon@localhost.localdomain>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use pkgAcquire::Run(50000) to make the pulseIntervall better suited
## DP: for interactive use

@DPATCH@
diff -urNad synaptic-0.56/common/rpackagelister.cc /tmp/dpep.JtjcAD/synaptic-0.56/common/rpackagelister.cc
--- synaptic-0.56/common/rpackagelister.cc	2005-02-24 11:13:13.000000000 +0100
+++ /tmp/dpep.JtjcAD/synaptic-0.56/common/rpackagelister.cc	2005-02-28 14:48:02.208559952 +0100
@@ -1268,7 +1268,7 @@
       return false;
 
    // Run it
-   if (Fetcher.Run() == pkgAcquire::Failed)
+   if (Fetcher.Run(50000) == pkgAcquire::Failed)
       return false;
 
 
@@ -1353,7 +1353,7 @@
    while (1) {
       bool Transient = false;
 
-      if (fetcher.Run() == pkgAcquire::Failed)
+      if (fetcher.Run(50000) == pkgAcquire::Failed)
          goto gave_wood;
 
       string serverError;
